crates/loader/src/assets.rs

```diff
@@ -76,18 +76,22 @@ pub(crate) fn component_dir(id: &str) -> String {
     // if the asset paths are also long. Longer term, consider an alternative approach where
     // we use an index or something for disambiguation, and/or disambiguating only if a clash is
     // detected, etc.
-    let id_sha256 = format!("{:x}", Sha256::digest(id));
+    let id_sha256 = bytes_sha256_digest_string(id.as_bytes());
     format!("{}_{}", UNSAFE_CHARACTERS.replace_all(id, "_"), id_sha256)
 }
 
+/// Return the hex-encoded SHA256 digest of the given bytes.
```

Comment:
Would it be worth moving these to a `digest.rs` or `sha256.rs` file instead of having them in `assets.rs`?  As they are used for bindling manifests and modules, they're not specific to assets.

Reply:
Done, thanks!